### PR TITLE
Add space ruin: abandoned engi sat

### PR DESCRIPTION
**Pull request by warriorstar-orion**
## What Does This PR Do

"This NT satellite, previously used by engineering departments of ships operating in the local sector, was abandoned after forecasts of extreme meteor activity. The satellite was evacuated and decommissioned, with all of its equipment relocated. Despite the actual damage from the meteors being minimal, the satellite has never been reclaimed by Nanotrasen."

The only things of value on the ruin are the hidden vodka and contraband posters, and the cell charger. No belts, no insuls, nothing. The suit storage units are empty, as are all of the equipment lockers. 

<details><summary>Notes for map reviewers</summary>

This DMM contains a lot of strange-looking varedits that I'm using to accentuate the feeling of a completely abandoned. Notably:

- Some light bulbs are broken in their brackets, requiring an `icon_state` and `status` change,
- All closets are base types that have subtype icons/data so that they are empty when spawned, rather than spawning with their typical equipment.
- A torn "SPACE" sign is just a renamed torn poster.

Additionally, the map uses modern airlock helpers. I set them to engineering for realism.

</details>

## Why It's Good For The Game
More space ruins are good, I thought this might be a cool nod to Para history, and bringing in an old NT property as a ruin makes it easy to incorporate into lore.

## Images of changes
In DMM:
2023_09_26__13_23_15__paradise dme  abandoned_engi_sat dmm  - StrongDMM

In-game, full dark,

In-game, APC powered

## Testing
Spawned ruin in, teleported, tested APC behavior, tested item interactions.

## Changelog
:cl:
add: Adds a space ruin, the abandoned NT Engineering satellite
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
A good homage to an awful time

**Comment by Bormac** [comment]
Needs cult references. Thats like 90% of what Engi sat was used for then 9% running from the nuke. 1% actual engi stuff. 
But really this is good.

File: code/datums/ruins/space_ruins.dm [hunk 1]
```diff
@@ -326,3 +326,11 @@
 	description = "A cargo shuttle in a wrecked condition. There are many unknown horrors in space and looks like its last crew has faced one of them."
 	allow_duplicates = FALSE
 	cost = 1 // With the loot it contains it shouldn't be found frequently
+
+/datum/map_template/ruin/space/abandoned_engi_sat
+	id = "abandoned_engi_sat"
+	suffix = "abandoned_engi_sat.dmm"
```
Comment (AffectedArc07) [inline code/datums/ruins/space_ruins.dm:332]: Yeah I've no idea why this is failing CI.

I checked this in the code and paths and its cased right.

Comment (warriorstar-orion) [reply code/datums/ruins/space_ruins.dm:332]: Whoops, fixed

**Comment by ATP-Engineer** [comment]
Add catwalk tiles on the interior maybe to look cooler?

**Review by Henri215 — APPROVED** [review]
LGTM

**Review by S34NW — COMMENTED** [review]
image
All your light buttons are one dir, even though they have dirs.

image
This is going to make me cry if not centered

image
Pipes under walls :( Although I don't see a better way so it will do here

image
Console faces a wall

image
console south facing

image
This one isn't an issue, but APCs autoname themselves based on their area so this var edit is kinda pointless

**Comment by warriorstar-orion** [comment]
> image All your light buttons are one dir, even though they have dirs.
> 

fixed

> image This is going to make me cry if not centered
>
fixed

> image Pipes under walls :( Although I don't see a better way so it will do here

i modernized most of the ruin but i figured in this case it's fine to have the pipes as they were on the original sat, before the mapping standards were really formulated

> image Console faces a wall
> 
> image console south facing
> 

both consoles fixed

> image This one isn't an issue, but APCs autoname themselves based on their area so this var edit is kinda pointless

fixed!

**Comment by Christasmurf** [comment]
beautifulasilostyou

**Review by S34NW — APPROVED** [review]

**Pull request merged**